lxqt-config-appearance/configothertoolkits.cpp

```diff
@@ -189,6 +218,24 @@ void ConfigOtherToolKits::setXSettingsConfig()
     }
 }
 
+void ConfigOtherToolKits::setGsettingsConfig(QString version, QString theme)
+{
+    updateConfigFromSettings();
+    if(!theme.isEmpty())
+        mConfig.styleTheme = theme;
+    QString commands;
+    if(version == QLatin1String("3.0"))
+        commands = getConfig(GTK3_GSETTINGS, ConfigOtherToolKits::GTK3GSETTINGS);
+    for(QString command : commands.split(QStringLiteral("\n"))) {
+        if(command.isEmpty() or command.startsWith(QStringLiteral("#")))
+            continue;
+        QStringList args = QProcess::splitCommand(QStringView(command));
+        bool ok = QProcess::startDetached(QStringLiteral("gsettings"), args);
+        if(!ok)
+            QMessageBox::critical((QWidget*) parent(), tr("Error"), tr("Error: gsettings can not be run"));
```

Comment:
1. If `gsettings` is not found, the error message needs to be shown only once and the loop should be ended. Here, it's shown with each loop cycle.
 2. A small typo: it's "Error: gsettings cannot be run" (you wrote "can not").

Reply:
Ok. Fixed.